domish: handle spaces in xmlns IRIs

In the "name" received from pyexpat only the last space delimits
the namespace from the tag/attr name.
Any previous spaces are part of the namespace IRI.

Signed-off-by: Michal Schmidt <mschmidt@redhat.com>

Index: twisted/words/xish/domish.py
===================================================================
--- twisted/words/xish/domish.py	(revision 29304)
+++ twisted/words/xish/domish.py	(working copy)
@@ -759,14 +759,14 @@
 
     def _onStartElement(self, name, attrs):
         # Generate a qname tuple from the provided name
-        qname = name.split(" ")
+        qname = name.rsplit(" ", 1)
         if len(qname) == 1:
             qname = ('', name)
 
         # Process attributes
         for k, v in attrs.items():
             if k.find(" ") != -1:
-                aqname = k.split(" ")
+                aqname = k.rsplit(" ", 1)
                 attrs[(aqname[0], aqname[1])] = v
                 del attrs[k]
 
